pkg/plugins/builtin/http/README.md

```diff
@@ -10,12 +10,27 @@ An action of type `http` requires the following kind of configuration:
 action:
   type: http
   configuration:
+    # mandatory
     url: http://example.org/user
+    # mandatory
     method: POST
+    # optional
+    timeout_seconds: 5
+    # optional
+    basic_auth:
+      user: test
+      password: test
+    # optional
+    deny_redirects: false
+    # optional
+    parameters:
+    - key: foo
+      value: bar
+    # optional
     headers:
     - name:  Authorization
       value: Basic {{.config.basicAuth}}
```

Comment:
change to:
  name: x-request-id
  value: xxx-yyy-zzz

Reply:
done.